lib/tasks/grda_warehouse.rake

```diff
@@ -294,6 +294,13 @@ namespace :grda_warehouse do
     rescue StandardError => e
       puts e.message
     end
+    begin
+      Rake::Task['driver:hmis:process_activity_logs'].invoke if ENV['ENABLE_HMIS_API'] == 'true'
+    rescue StandardError => e
+      puts e.message
+      Sentry.capture_exception(e)
```

Comment:
The other tasks just rescue exceptions; I guess there must be another mechanism for reporting these exceptions?

Reply:
I don't know, I had the same question. @eanders ?

Reply:
I'm going to add calls to sentry for the other cases. If it ends up being too noisy we can always remove it